src/ccl_massfunc.c

```diff
@@ -300,13 +378,38 @@ static double massfunc_f(ccl_cosmology *cosmo, double halomass, double a, double
     return NAN;
   }
 }
-static double ccl_halo_b1(ccl_cosmology *cosmo, double halomass, double a, double odelta, int * status)
+
+static double ccl_halo_b1(ccl_cosmology *cosmo, double halomass, double a, double odelta, int *status)
 {
-  double fit_A, fit_B, fit_C, fit_a, fit_b, fit_c, overdensity_delta, y;
+  double fit_A, fit_B, fit_C, fit_a, fit_b, fit_c, fit_p, overdensity_delta, y;
   double delta_c_Tinker, nu;
   double sigma=ccl_sigmaM(cosmo,halomass,a, status);
   switch(cosmo->config.mass_function_method) {
 
+  // Equation (12) in  arXiv: 9901122
+  // Derived using the peak-background split applied to the mass function in the same paper
+  // Note that Sheth & Tormen (1999) use nu=(dc/sigma)^2 whereas we use nu=dc/sigma
+  // TODO: Mead: Somehow enforce that this should only work with a virial-density Delta_v
```

Comment:
TODO: Does this need to be addressed?

Reply:
It had already been addressed. Only the comment remained. Done